recipes/scikit-digital-health/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "scikit-digital-health" %}
+{% set version = "0.9.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PfizerRD/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 8adc61af12e92e047a06f276a32a56cdf4deca350f8bb24df95c25ea177fc749
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ compiler('m2w64_c') }}  # [win]
+    - {{ compiler('m2w64_fortran') }}  # [win]
+  host:
+    - python
+    - pip
+    - numpy
+    - scipy >=1.3.1
```

Comment:
Probably everything below this line is not required at build time unless the package imports itself to get the version ro something. Do you mind confirming that before we merge?

Reply:
Do python, (pip?), numpy, gsl, and libzip not need to be there as well? they are all used in the build (numpy's distutils & C-API), and the shared libraries are linked to.

Reply:
pip, numpy yes. GSL and libzip probably, we need to check if the package uses them to link or use their headers. Things seems to be passing. Let me take a second look...